src/tests/src/solver/simulation/test-hydro-final-reservoir-level-functions.cpp

```diff
@@ -101,161 +108,133 @@ struct Fixture
     Area* area_2;
 };
 
-BOOST_FIXTURE_TEST_SUITE(s, Fixture)
+BOOST_FIXTURE_TEST_SUITE(final_level_validator, Fixture)
 
-BOOST_AUTO_TEST_CASE(initialize_modifier_for_area_1___modifier_is_applicable)
+BOOST_AUTO_TEST_CASE(all_parameters_good___check_succeeds_and_final_level_is_usable)
```

Comment:
Unit tests were adapted for new class **FinalLevelValidator**;
Consequently Some tests become unnecessary, and disappeared.